projects/sandbox/pyproject.toml

```diff
@@ -16,14 +17,14 @@ datadir = "${DATA_DIR}"
 logdir = "${BASE_DIR}/log"
 sample_rate = 2048
 ifos = ['H1', 'L1']
-glitch_start = 1262607622
-train_start = 1262607622
-train_stop = 1262620622
-test_stop = 1262751694  
+glitch_start = 1262653854
+train_start = 1262653854
+train_stop = 1262686622 # 32768 s segment
+test_stop = 1262945822 # 3 day window for test
 channel = "DCS-CALIB_STRAIN_CLEAN_C01"
 frame_type = "HOFT_C01"
 state_flag = "DCS-ANALYSIS_READY_C01:1"
-resnet = {layers = [3, 3, 3]}
+resnet = {layers = [2, 2, 2, 2], norm_groups = 8}
```

Comment:
My guess is this is set to `> n_ifos` to account for future layers where more channels are introduced?

Reply:
So the norm layer I've built will use `min(in_channels, norm_groups)`, so for the first layer this will equal to `num_ifos`, and then `norm_groups` after that. The number of groups to use for the later layers is a hyperparameter that will need to be tuned, this is just a random first guess that seems to work decently well.